README.md

```
@@ -1,358 +1,182 @@
 
-# Cursor
-mikk mcp install --tool cursor
+---
 
-# Windsurf (this IDE)
-mikk mcp install --tool windsurf
+## How It Works
 
-# VS Code
-mikk mcp install --tool vscode
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)